<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPreset</class>
 <widget class="QDialog" name="DialogPreset">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config Presets</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>8</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_Label_7</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>4</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_Label_3</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Label_1"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>3</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_Label_2</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_Point">
     <property name="text">
      <string>Point</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Label_7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Label_4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="spinBox_Point_1">
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>2</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_Label_1</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Label_6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QSpinBox" name="spinBox_Point_7">
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_Button">
     <property name="text">
      <string>Button</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Label_0"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>6</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_Label_5</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Label_8">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_Label">
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QSpinBox" name="spinBox_Point_5">
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>7</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_Label_6</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QSpinBox" name="spinBox_Point_8">
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
   <item row="12" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QSpinBox" name="spinBox_Point_4">
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinBox_Point_0">
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QSpinBox" name="spinBox_Point_6">
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Label_5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Label_3"/>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="spinBox_Point_2">
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>9</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_Label_8</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="spinBox_Point_3">
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>5</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_Label_4</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Label_2"/>
   </item>
   <item row="11" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_0">
     <property name="text">
      <string>1</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_Label_0</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QPushButton" name="pushButton_Default">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Label_0</tabstop>
  <tabstop>spinBox_Point_0</tabstop>
  <tabstop>lineEdit_Label_1</tabstop>
  <tabstop>spinBox_Point_1</tabstop>
  <tabstop>lineEdit_Label_2</tabstop>
  <tabstop>spinBox_Point_2</tabstop>
  <tabstop>lineEdit_Label_3</tabstop>
  <tabstop>spinBox_Point_3</tabstop>
  <tabstop>lineEdit_Label_4</tabstop>
  <tabstop>spinBox_Point_4</tabstop>
  <tabstop>lineEdit_Label_5</tabstop>
  <tabstop>spinBox_Point_5</tabstop>
  <tabstop>lineEdit_Label_6</tabstop>
  <tabstop>spinBox_Point_6</tabstop>
  <tabstop>lineEdit_Label_7</tabstop>
  <tabstop>spinBox_Point_7</tabstop>
  <tabstop>lineEdit_Label_8</tabstop>
  <tabstop>spinBox_Point_8</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogPreset</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogPreset</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
